### PR TITLE
Rename "Commercial Support" to "Paid Support" #80

### LINKED ISSUE #80: Rename "Commercial Support" to "Paid Support"
Commerce/Commercial is associated with large scale exchanges, lets be plain speaking on this front. We intend to offer "Paid" support via an "Incident-based Support subscription" membership Tier at our Open Collective:
https://opencollective.com/the-rockstor-project

This issue should also rename the link involved:
https://rockstor.com/commercial_support.html
and ensure the old link, embedded within our older Web-UI's, redirects to the new link of say:
https://rockstor.com/paid_support.html

Hugo "Aliases" should take care of this redirect:
https://gohugo.io/content-management/urls/#aliases

### PR DESCRIPTION
**Pull request by phillxnet**
Simplify/update wording regarding our planned relaunch of the 'Incident-based Support subscription'. Present as a "Support included" variant of our existing 'Stable Updates subscription' membership tier.

Incidental changes:
- Proposed relaunch cost revision.
- Update website copyright notice.
- Add Hugo page alias re our prior "/commercial_support.html" URL.

Fixes #80

### CODE REVIEW

**Comment by phillxnet** [comment]
# Notes on proposed relaunch cost revision

The more current thoughts on this from me are that we incorporate a Stable Updates subscription within any support tier that we offer. This way we avoid the complexities, on our and other's sides, of having to subscribe to both a stable Updates subscription and an incident based support tier. Ergo we have say a €240 / year single instance support tier that includes a Stable Update subscription.

€24 - Stable Updates subscription membership
€240 - Same as €24 option but with single instance support included.

So the equivalent of a €216 premium for single incident paid support. In the context that we are a non-business non-profit and small !!

This way Appman would treats each near identically. We may need an additional flag (order.field) to indicate if an incident has been used, and we show this element within Appman's Web-UI only if this is a support tier variant. Upon yearly renewal (New Transaction updating existing order) we ensure this order.field is re-set to what-ever number of support incidents the specific tier level represents.

Pricing is based on BareOS https://www.bareos.com/ from a little while back.

https://www.bareos.com/pricing/

From the details we have: https://www.bareos.com/Bareos_Pricelist.pdf
They also have a subscription as requirement - but we bundle this for the above reasons re Appman integration and user/back-end simplicity.

- BASP-Q Basic 3 not guaranteed 1.080 € (... max. 15 clients and does not cover plugins.)
- BASP-S Standard 10 2 business days 2.600 €

So our current equivalent of 48 hours response would be their €2600/10 = €260/incident
Or their €1080/3 = €360/incident (but note this has no service level guarantee !!)
So for business clients our €240 with best effort at 48 hour response, we should be within current context: reasonable. And this would be the recommended/suggested option for small-to-medium businesses.
Expiration after 1 year if unused (within reason).

**Comment by phillxnet** [comment]
# Caveat re Draft

When testing this PR, the intended page/menu changes look to be as intended. But we appear now to have a broken downloads page! I.e. no entries for each of the Downloads available - only the headers are now listed !!

@FroggyFlox or @Hooverdan96 any ideas on this. I've made no changes in related areas from what I can tell  !!
I'll update here if I track this one down.

**Comment by phillxnet** [comment]
Moved status to 'ready for review' as our back-end has now received a work-around re our upstream Hugo update sensitivity detailed in the last comment.

File: content/paid_support.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+---
+
+## Support FAQ
+{{< bootstrap-table table_class="table table-bordered border-primary table-striped" >}}
+
+Question | Answer
+--- | ---
+**How much do you charge for Rockstor support?** | Support is charged on a per incident basis, expected relaunch price €240 + tax (if applicable) per incident.
+**How is an incident defined?** | An incident is defined as a question relating to a specific, discrete issue, and may involve several interactions with support prior to that issue's resolution.
+**What is not an incident?** | If your issue is caused by a Rockstor package software bug then there is no charge. In this case we prioritising the related bugfix.
```
Comment (Hooverdan96) [inline content/paid_support.md:58]: **What is not an incident?** | If your issue is caused by a Rockstor package software bug then there is no charge. In this case we will be prioritising the related bugfix.

Comment (Hooverdan96) [inline content/paid_support.md:58]: **What is not an incident?** | If your issue is caused by a Rockstor package software bug then there is no charge. In this case we will be prioritising the related bugfix.

File: content/paid_support.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+**How do I contact you for paid support?** | Email support with your Open Collective 'Incident-based Support subscription' membership tier order number.
```
Comment (Hooverdan96) [inline content/paid_support.md:59]: Maybe add the support email address in here, even though it's mentioned in other places.

**Comment by Hooverdan96** [comment]
Two small comments on the page text ...

File: content/paid_support.md (hunk 1 above)
Comment (phillxnet) [reply content/paid_support.md:58]: @Hooverdan96 
Thanks, I'd missed that entirely. Fancy popping in as a suggestion on the relevant line so we have attribution?

File: content/paid_support.md (hunk 2 above)
Comment (phillxnet) [reply content/paid_support.md:59]: If I do, it doesn't show as a link, only text. I tried initially popping in our base-of-page email link but the way we do this table does not allow that kind of rich content. I'll have another look thought.

**Comment by phillxnet** [comment]
@Hooverdan96 Thanks for the suggestions. I'll take another pass and see what it looks like then.

File: content/paid_support.md (hunk 1 above)
Comment (Hooverdan96) [reply content/paid_support.md:58]:
```suggestion
**What is not an incident?** | If your issue is caused by a Rockstor package software bug then there is no charge. In this case we will be prioritising the related bugfix.
```

**Comment by phillxnet** [comment]
I've done another pass and incorporated @Hooverdan96 suggestion, it does mean we now have a couple of rouge hard-coded email links (markdown), but it works from the user perspective and I've added an comment to note these in config.toml where all other emails references come from. That boostrap table addition is a tricky one via the shortcode.

I've also done some more simplification/reduction word-wise which I think helps.

If there are no more suggestions I'll merge and publish this on the 17th.

**Pull request merged**